js/exporter/exceljs/export_format.js

```diff
@@ -117,6 +117,39 @@ function _convertNumberFormat(format, precision, currency) {
     return result;
 }
 
+function _hasCSVInjection(value) {
+    if(!value || value.length < 2) { return false; }
+
+    return _includesCSVExpression(value);
+}
+
+function _hasCSVQuotedInjection(value, textQualifier) {
+    if(!value || value.length < 4 || value[0] !== textQualifier) {
+        return false;
+    }
+
+    return _includesCSVExpression(value.substring(1, value.length - 1));
+}
+
+function _includesCSVExpression(value) {
+    const injectionPrefix = /^[@=\t\r]/;
+    const injectionPrefixSigns = /^[+-]/;
```

Comment:
As far as i remember i suggested another naming :)

Reply:
Yes. It is not a typo. I suggest this name more suitable in this case.

Reply:
https://github.com/DevExpress/DevExtreme/pull/23600/commits/2a70f9550a42b1a980f8794c25e20c390520b090